docs/runbooks/VirtControllerDown.md

```diff
@@ -0,0 +1,24 @@
+# VirtControllerDown
+
+## Meaning
+No running virt-controller was detected for the last 5 min. Virt-controller deployment has a default replica of 2 pods. 
+
+## Impact
+Complete failure in VM lifecycle management i.e. launching a new VM instance or shutting down an existing VM instance.
+
+## Diagnosis
+
+- Set the environment variable NAMESPACE
+
+    ```
+    export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+    ```
+
+- Observe the status of the virt-controller deployment
+    ```
+    kubectl get deployment -n $NAMESPACE virt-controller -o yaml
+    ```
+Observe the logs of the kubecontroller manager pod, to see why it cannot create the virt-controller pods.
```

Comment:
@iholder-redhat @rmohr  What command should they run here

Reply:
@iholder-redhat @rmohr How do they observe the logs of the kubecontroller manager pod? What are the steps they should run?

Reply:
`k get pods -n kubevirt` (assuming kubevirt components reside in `kubevirt` namespace. Also `k get pods -A` is possible to get all pods from all namespaces).

Then, `k get logs <virt-launcher-pod>`. virt-laucnher pod should have name similar to `virt-controller-7888c64d66-dzc9p`

Reply:
p.s. there could be several pods that run virt-controller

Reply:
@iholder-redhat Thank you. I updated the runbook. Please review.

Reply:
looks good to me